ui/app/templates/components/calendar-widget.hbs

```diff
@@ -59,46 +57,72 @@
     {{#if this.showCalendar}}
       <div class="calendar-widget-container" data-test-calendar-widget-container>
         <div class="select-year">
-          <Hds::Button
-            data-test-previous-year
-            id="previous-year"
-            disabled={{this.disablePastYear}}
-            {{on "click" this.subYear}}
-            @color="secondary"
-            @text="previous year"
-            @icon="chevron-left"
-            @isIconOnly={{true}}
-          />
-          <p data-test-display-year>
+          {{#if this.disablePastYear}}
+            <Hds::TooltipButton
```

Comment:
The HDS tooltip looks slightly different than our current one - are we okay with the style discrepancy? Originally we had said we weren't. 

Updating all tooltips seems outside scope - one option is updating the color/border of the strucutre tooltip to match this style?

Reply:
This particular original tooltip wasn't even working, that's why I updated it. But if we'd rather do all the Tooltips at once I can revert this

Reply:
I was able to update the structure tooltip styling! 🎉 good idea

<img width="1327" alt="Screenshot 2023-12-13 at 5 37 01 PM" src="https://github.com/hashicorp/vault/assets/82459713/a24f94ca-608e-4c63-93c2-4753681bdd7d">